### DISCUSSION TITLE
Issue in running dockerized version of open5GS on ubuntu (22.04, Jammy)

### DISCUSSION THREAD

**Discussion by amitpalkar** [srsran/srsRAN_Project#662] [General Help]
hi, i am trying to run the dockerized version of open5GS on ubuntu (22.04, Jammy) via the command docker compose up --build 5gc
However i am seeing the below error
82.40 Setting up nodejs-doc (12.22.9~dfsg-1ubuntu3.5) ...
82.42 Processing triggers for libc-bin (2.35-0ubuntu3.8) ...
82.47 /bin/sh: 1: npm: not found
failed to solve: process "/bin/sh -c curl -fsSL [https://deb.nodesource.com/setup_${NODE_MAJOR}.x](https://deb.nodesource.com/setup_$%7bNODE_MAJOR%7d.x) | bash - && apt-get install -y nodejs && cd open5gs/webui && npm ci --no-optional" did not complete successfully: exit code: 127

My Docker version is Docker version 26.1.4, build 5650f9b. Can i know where is the issue and how can i resolve the issue?

Regards,
Amit Palkar

**Comment by pgawlowicz** [comment]
@amitpalkar any update on this issue? could you remove the old container/image and build again?

**Comment by amitpalkar** [comment]
@pgawloxwicz, Apologies for Late reply.  It is working.  There was some permission issues on the system for downloading files.  Once the permissions were granted it started working
